app/javascript/js/controllers/clipboard_controller.js

```diff
@@ -0,0 +1,20 @@
+import Clipboard from '@stimulus-components/clipboard'
+
+export default class extends Clipboard {
+  static targets = ['icon', 'iconCopied']
+
+  connect() {
+    super.connect()
+  }
+
+  copied() {
+    this.iconTarget.classList.add('hidden')
+    this.iconCopiedTarget.classList.remove('hidden')
+
+    // Reset the icon after a 2 seconds delay
+    setTimeout(() => {
+      this.iconTarget.classList.remove('hidden')
+      this.iconCopiedTarget.classList.add('hidden')
+    }, 2000)
```

Comment:
I think this can be configured with `data-clipboard-success-duration-value` 

From: https://www.stimulus-components.com/docs/stimulus-clipboard

Reply:
It is not that easy, It works with  **data-clipboard-success-content-value** (this work only with plain text) , we want to show another object so we can not probably use **data-clipboard-success-duration-value**. I tried use that but it do not work.